src/coverage.ts

```diff
@@ -71,6 +71,12 @@ export async function runCoverage(skipFiles: string[], compileCommand: string, t
     // wait for the tests to finish
     await once(forked, 'close');
 
+    // Clean up before writing report
+    cleanUp();
+    exitHook(() => {
+      // do nothing.
+    });
```

Comment:
This doesn't invalidate the previous `exitHook` call. `cleanUp` will still run. We need to make sure it's safe to call twice. I think it already is.

Reply:
I followed this https://github.com/sindresorhus/exit-hook#usage. Is it `exitHook(() => {})()` to remove previous `exitHook`?

But yeah, it is safe to be called twice so it is fine.

Reply:
I removed that. If there is a way to invalidate previous `exitHook` we can add that in.

Reply:
Oh, the return of exitHook will give you a function to remove that when called. So it is:

```
const cancelExitHook = exitHook(cleanUp);

// ...

cleanUp()
cancelExitHook()
```

@frangio do you think it is necessary or just let the `cleanUp` runs twice.

Reply:
I would just let it run twice.